pandas/core/frame.py

```diff
@@ -4070,7 +4070,11 @@ def _apply_standard(self, func, axis, ignore_failures=False, reduce=True):
         # e.g. if we want to apply to a SparseFrame, then can't directly reduce
         if reduce:
 
-            values = self.values
+            # The 'values' attribute of a pandas Series
+            # is a numpy ndarray, so the dtype will be
+            # guaranteed to be valid when passed into
```

Comment:
no, instead of the where ``np.empty`` is used, use ``pandas.core._sanitize_array.create_from_value`` (prob should strip that out into a private helper function). this provides compat for things like this.